backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/OktaRepository.java

```diff
@@ -32,6 +32,9 @@
 
   public Map<String, OrganizationRoleClaims> getAllUsersForOrganization(Organization org);
 
+  public void createOrganization(
```

Comment:
Here or in the LiveOktaRepository implementation of this method would seem like it would have been a good place to explain what happens when `migration` is `true`. The code isn't unreasonably difficult to read (subject to the constraints of the API) but it's nice to be able to cross-reference against a block comment when trying to figure out if it does what it's supposed to.